refresh/templates/extensions/WatsonConversationExtension.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import ConversationV1
+import CloudFoundryConfig
+
+extension Conversation {
+
+   public convenience init(service: WatsonConversationService) {
+
+        let version = "<%- service.version %>"
```

Comment:
There was a bug with this line not getting run through a template - did it get fixed? I think we need to fix this before we can release @jkingoliver